Samples/WebSignIn/README.md

```diff
@@ -0,0 +1,30 @@
+# WebSignIn Samples
+
+This sample contains three separate sample application schemes:
+
+## WebSignIn (iOS)
+
+This sample demonstrates web authentication using the WebAuthenticationUI library.  To get started, open the `Okta.plist` file and update its contents with your application's configuration settings to run the sample.
+
+ Key | Required | Description |
+---|---|---
+`issuer` | ✔ | Issuer URL for the client.
+`clientId` | ✔ | Client ID for the Okta application.
+`scopes` | ✔ | Scopes the client is requesting.
+`redirectUri` | ✔  | Redirect URI for the Okta application.
+`logoutRedirectUri` | | Logout URI used for the Okta application.
+Other... | | Any additional keys will be passed to the `additionalParameters` argument of the initializer.
```

Comment:
Per above, I'm assuming this is the readme for WebSignIn (iOS). That implies something like:

## Abstract

Use the Web Authentication UI library from this SDK to add sign-in to your app.

## Configure the App

Update the `Okta.plist` file with the information for your Okta Org Application Integration. The following table shows how to configure each key and whether the key is required:

 Key | Required | Description |
 ---|---|---
 `issuer` | ✔ | The domain of your registered Okta org followed by `/oauth2/default`, such as `https://dev-1234567.okta.com/oauth2/default`.
  `clientId` | ✔ | The client ID from the Application Integration in the Okta Admin console, such as `0ux3rutxocxFX9xyz3t9`.
  `scopes` | ✔ | A space-delimited list of the permissions, or OAuth scopes requested by the client. The existing list may not need updating.
  `redirectUri` | ✔  | Redirect URI for the Okta application, such as `com.okta.1234567:/callback`.
  `logoutRedirectUri` | | Logout URI used for the Okta application, such as `com.okta.1234567:/logout`

Any other keys and values that you add to the plist are passed to the `additionalParameters` argument of the initializer.

> NOTE: be clear what the `additionalParameters argument is... is it part of the webauthui lib? part of OAuth?